source/gloperate-glfw/source/Window.cpp

```diff
@@ -472,25 +471,25 @@ void Window::handleEvent(WindowEvent & event)
 
 void Window::clearEventQueue()
 {
-    while (!m_eventQueue.empty())
-    {
-        delete m_eventQueue.front();
-        m_eventQueue.pop();
-    }
+    std::queue<std::unique_ptr<WindowEvent>> empty;
```

Comment:
I found it [here](http://stackoverflow.com/questions/709146/how-do-i-clear-the-stdqueue-efficiently).